src/main/java/org/prebid/server/currency/proto/response/CurrencyConversionRates.java

```diff
@@ -0,0 +1,21 @@
+package org.prebid.server.currency.proto.response;
```

Comment:
Generally we create sub-packages for request and response models if there is considerable amount of each. In this particular case this is not needed so this POJO could be placed in `org.prebid.server.currency.proto`